tests/test_watson.py

```diff
@@ -21,10 +20,7 @@
         )
     ) / 'resources'
 
-if not PY2:
-    builtins = 'builtins'
-else:
-    builtins = '__builtin__'
+builtins = 'builtins'
```

Comment:
I think we don't need this var anymore, we can use `'builtins'` directly where we need.

Reply:
Done.